src/java/org/apache/cassandra/db/virtual/VirtualKeyspaceRegistry.java

```diff
@@ -73,6 +76,11 @@ public TableMetadata getTableMetadataNullable(TableId id)
         return null != table ? table.metadata() : null;
     }
 
+    public ImmutableSet<String> getKeyspaces()
+    {
+        return ImmutableSet.copyOf(virtualKeyspaces.keySet());
+    }
+    
```

Comment:
this is not needed, as we can get it from `SchemaConstants.isVirtualSystemKeyspace(String)`.